actionpack/lib/action_dispatch/journey/gtg/builder.rb

```diff
@@ -131,10 +134,6 @@ def build_followpos
                 lastpos(n.left).each do |i|
                   table[i] += firstpos(n.right)
                 end
-              when Nodes::Star
-                lastpos(n).each do |i|
-                  table[i] += firstpos(n)
-                end
```

Comment:
I want to double check this, but I found that this was the correct pattern for a regexp star (match zero of more) but not a rails-param star (typically match 1 or more), but this may need to check against `nullable`